src/components/Garden/Preferences/EVMExecutor.js

```diff
@@ -229,12 +231,12 @@ function EVMExecutor({ evmcrispr }) {
     code,
   ])
 
-  const handleOnContractAddressChange = useCallback(event => {
+  const handleOnContractAddressChange = useCallback((event) => {
```

Comment:
Just a comment, that you dont need to use useCallback for a simple function like this. In my opinion since is a simple setState, you can do it in the onChange itself, `onChange={(e) => setExternalContractAddress(e.target.value)}`

Reply:
useCallback is better performance against anonymous function on onChange.
Plus its better to future changes too.

just my 2 honeys on that.

Reply:
You don't need to memoize a setState

Reply:
I agree with you @tmoutinho - but with that reasons https://dmitripavlutin.com/dont-overuse-react-usecallback/

Reply:
We are using useCallback every time, and it's not needed, thats my point